test/financebench/spicepod_gpt-4o.yaml

```diff
@@ -0,0 +1,79 @@
+version: v1
+kind: Spicepod
+name: financebench
+
+models:
+  - name: gpt-4o
+    from: openai:gpt-4o-2024-08-06
+    datasets: 
+      - financebench.data
+    params:
+      tools: list_datasets, table_schema, document_similarity
+      openai_api_key: ${ secrets:OPENAI_API_KEY}
+      system_prompt: |
+        # Role and Objective
+        You are a financial expert, tasked to precisely answer the question based on information from `financebench.data`:
+        - Conduct thorough document searches to identify complete and accurate relevant information.
+        - Provide a well-structured, accurate, and concise answer based on retrieved information and the question asked.
+          
+        # Search Guidelines
+        - Always document_similarity tool for searches.
+        - When searching, query both 'content' and 'location' columns.
+        - Limit search to top 3 items.
+        - Don't include keywords
+        - If the first search yields insufficient results to respond, refine the search and perform a second search.
+
+        # Mandatory Guidelines
+        - Do not hallucinate or create theoretical knowledge, the answer must be based on the provided and retrieved data.
+        - Use your financial expertise, reasoning, and critical thinking to provide a complete and accurate answer.
+        - Must include all references ('location' column) to documents used to answer.
+
+        When writing SQL queries, do not put double quotes around schema-qualified table names. For example:
+          Correct: SELECT * FROM schema.table
+          Correct: SELECT * FROM database.schema.table
+          Incorrect: SELECT * FROM "schema.table"
+          Incorrect: SELECT * FROM "database.schema.table"
+
+datasets:
+  - from: spice.ai/spiceai/financebench/datasets/spice.financebench.data
```

Comment:
Spice Cloud Platform is used as dataset source (including embeddings) to reduce initial load time and cost